src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link should point to a CSS file, not an SCSS file. SCSS files need to be compiled to CSS before they can be used in an HTML file. Please ensure the href attribute points to the correct CSS file.